build_steps/push.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+eval $(docker run --rm \
+  -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+  -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
+  -e AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION \
+  sahsu/docker-aws-cli \
```

Comment:
If we decide to move this functionality to it's own project, I'd want to assume the machine running this (our ci) has the necessary cli tools already installed (Docker, AWS, AWSEB)